waifus/mayuri shiina.json

```diff
@@ -0,0 +1,11 @@
+﻿{
+	"type" : 0,
+	"japanese-name" : "椎名 まゆり",
+	"english-name" : "Mayuri Shiina",
+	"year" : 1994,
+	"month" : 2,
+	"day-of-month" : 1,
+	"age-in-show": 16,
+	"image" : "https://vignette.wikia.nocookie.net/steins-gate/images/0/0e/Mayuri_Shiina.png/revision/latest?cb=20181024172638",
```

Comment:
I think this one is better https://vignette.wikia.nocookie.net/steins-gate/images/a/ac/Mayuri_full_profile.png